src/main/drivers/sound_beeper.h

```diff
@@ -20,13 +20,18 @@
 #ifdef BEEPER
 #define BEEP_TOGGLE              systemBeepToggle()
 #define BEEP_OFF                 systemBeep(false)
+#ifdef USE_DSHOT
+#define BEEP_ON                  systemBeepDshot()
```

Comment:
This is wrong. Even if Dshot is compiled, beeping should be done with the beeper by default. This will break beeping for just about all boards if Dshot is not used as the motor control protocol.